Automattic/wp-calypso
# Add/social instagram connection

This change adds the Instagram connection to Calypso.

## Proposed Changes

* Added the new connection handlers
* Added service description and a new badge
* Added service example

## Testing Instructions

* Check out this PR
* Check the following change on wpcom: D111103-code. Sandbox the WP API.
* Go to `/marketing/connections/<your_site>` URL, and look for the new Instagram connection.
  * Description and looks should match with the designs
  * It should have a new badge
  * You should be able to connect an Instagram account, it should appear

The order comes from WPCOM, that needs to be changed there so we have instagram second

<img width="875" alt="CleanShot 2023-05-23 at 10 59 19 png" src="https://github.com/Automattic/wp-calypso/assets/36671565/37308afc-9f3e-49a1-a5ad-88f2cb3da992">

## Pre-merge Checklist
